desktop/core/base_requirements.txt

```diff
@@ -26,6 +26,7 @@ django-webpack-loader==1.0.0
 djangomako==1.3.2
 djangorestframework-simplejwt==5.2.1
 djangorestframework==3.14.0
+djangosaml2==1.9.3
```

Comment:
@amitsrivastava - Do we really need it here given we now depend on `generate_requirement.py`? Will the base file remain after the cleanup?

Reply:
I think we probably do not need base_requirements.txt, @ranade1 should be able to comment. I added the dependency here only to keep things in sync. But, you are right we should work on clean-up and remove this file if not needed.